### DISCUSSION TITLE
I'm new to this, how do i tell the bot to run other stores?

### DISCUSSION THREAD

**Discussion by xqawas** [jef/streetmerchant#2622] [Help]
The files for the stores are all there, but the bot just runs BestBuy and Amazon.com. I'm really confused and need help please.
Or how can i tell the bot to only run PS5's

**Comment by c0rrre** [comment]
You can add stores and other filters (like searching for ps5 only) with this parameters:

https://www.jef.buzz/streetmerchant/reference/filter/#supported-stores

**Comment by xqawas** [comment]
Thank you so much.
I assume i can just put all the variables next to eachother like:
nvidia-fr, nvidia-de, nvidia-gb
for their respective argument
